apps/aecore/lib/aecore/chain/header.ex

```diff
@@ -7,6 +7,12 @@ defmodule Aecore.Chain.Header do
   alias Aeutil.Bits
   alias Aecore.Keys.Wallet
 
+  @txs_hash_size 32
+  @header_hash_size 32
+  @root_hash_size 32
+  @pow_size 168
+  @pubkey_size 33
```

Comment:
could you extract the other magic numbers in this file as well

Reply:
ok